api/app/graphql/types/query_type.rb

```diff
@@ -2,5 +2,10 @@ module Types
   class QueryType < Types::BaseObject
     # Add root-level fields here.
     # They will be entry points for queries on your schema.
+
+    field :current_user, Types::UserType, null: true
+    def current_user
+      context[:current_user]
+    end
```

Comment:
At scale, is there a better way to define queries on the base `QueryType` class? (maybe like how MutationsType works)

Reply:
No.  This is one of the convention things with graphql-ruby, they used to have a more js like syntax.  Good/bad/indifferent thought it does allow you to follow the more "ruby" class like api, which gives you some easy test hooks and also allows graphql ruby to hook into that class system in order to do nice things with the schema.   Mostly just follow the convention here as much as possible and things will be easier.